x11-libs/libXaw3dXft/libXaw3dXft-1.6.2h.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xorg-3
+
+DESCRIPTION="Xaw3dXft library"
+HOMEPAGE="https://sourceforge.net/projects/sf-xpaint"
+SRC_URI="mirror://sourceforge/sf-xpaint/${P}.tar.bz2"
+
+KEYWORDS="~amd64 ~x86"
+IUSE="unicode xpm"
+
+RDEPEND="
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXft
+	x11-libs/libXmu
+	x11-libs/libXt
+	xpm? ( x11-libs/libXpm )"
+DEPEND="${RDEPEND}
+	x11-base/xorg-proto"
+BDEPEND="
+	sys-devel/flex
+	virtual/yacc"
+
+src_configure() {
+	local XORG_CONFIGURE_OPTIONS=(
+		$(use_enable unicode internationalization)
```

Comment:
This should probably be forced on already.

Reply:
`netpbm` was indeed a new dependency, but I don't understand the problem with `epstopdf: command not found`

Neither *xpaint* nor *libpgf* require `epstopdf`.  I even tried removing the executable from my system and it still compiles fine.  I see `doxygen` requires `epstopdf`, but it also installs it as a dependency, so I don't understand how to get that error, since **USE=doc** requires `doxygen`.

Reply:
I checked.  `--enable-internationalization` is the default.  With **USE=-unicode**, that will pass `--disable-internationalization` to configure, which is probably what we want.